app/assets/stylesheets/index.scss

```diff
@@ -1,67 +1,64 @@
+.nav {
+  background: rgb(33, 90, 90);
+  height: 50px;
+}
+
 .container2 {
   width: 100%;
   height: 100%;
   background-color: rgb(107, 106, 106);
   display: grid;
-  grid-column-gap: 25px;
-  grid-template-columns: repeat(6, 1fr);
-  grid-template-rows: 0.25fr repeat(3, 1fr);
-  grid-template-areas:
-    "nav nav nav nav nav nav"
-    "box1 box1 box2 box2 box2 box2"
-    "box1 box1 box2 box2 box2 box2"
-    "box1 box1 box3 box3 box4 box4"
-    "box1 box1 box3 box3 box4 box4";
-}
+  grid-gap: 25px;
+  grid-template-columns: repeat(7, 1fr);
+  grid-template-rows: repeat(3, 1fr);
 
-.nav {
-  grid-area: nav;
-  background: rgb(33, 90, 90);
-}
-
-.box1 {
-  grid-area: box1;
-  background: rgb(97, 14, 97);
-  flex-direction: column;
-  display: flex;
-  text-align: center;
-  img {
-    width: 40px;
-    height: 40px;
-    margin: 5rem auto 2rem auto;
+  .box1 {
+    grid-column: 1;
+    grid-row: span 3;
+    background: rgb(97, 14, 97);
+    flex-direction: column;
+    display: flex;
+    text-align: center;
+    img {
+      width: 40px;
+      height: 40px;
+      margin: 5rem auto 2rem auto;
+    }
   }
-}
 
-.box2 {
-  grid-area: box2;
-  background: darkblue;
-  margin-top: 25px;
-  margin-right: 25px;
-}
+  .box2 {
```

Comment:
I would probably name these thing more specifically. So generic and gives me no other information than this is one "box" with the number 1; so I might expect more. Why not call it "loginBox" or "beltProgression" or whatever relevant? ;)